spec/features/admin/dashboard_spec.rb

```diff
@@ -0,0 +1,81 @@
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+
+      user_1 = create(:user, role: 0)
+      user_2 = create(:user, role: 0)
+      user_3 = create(:user, role: 0)
+
+      order_1 = create(:order, created_at: Date.today, user: user_1, status: 3) # cancelled
+      order_2 = create(:order, created_at: Date.today, user: user_2, status: 1) #packaged
+      order_3 = create(:order, created_at: Date.today, user: user_3, status: 2) # shipped
+      order_4 = create(:order, created_at: Date.today, user: user_1, status: 0) # pending
+
+      item_1 = create(:item)
+      item_2 = create(:item)
+      item_3 = create(:item)
+      item_4 = create(:item)
+      item_5 = create(:item)
+
+      merchant_1 = create(:merchant)
+      merchant_2 = create(:merchant)
+      merchant_3 = create(:merchant)
+
+      merchant_1.items << [item_1, item_2]
+      merchant_2.items << [item_2, item_3, item_4]
+      merchant_3.items << [item_3, item_5]
+
+      merchant_1_employee = create(:user, role: 1, merchant: merchant_1)
+      merchant_2_employee = create(:user, role: 1, merchant: merchant_2)
+      merchant_3_employee = create(:user, role: 1, merchant: merchant_3)
+
+      order_1.item_orders.create(item: item_1, quantity: 10, price: item_1.price)
+      order_1.item_orders.create(item: item_2, quantity: 10, price: item_2.price)
+      order_2.item_orders.create(item: item_1, quantity: 10, price: item_1.price)
+      order_2.item_orders.create(item: item_3, quantity: 10, price: item_3.price)
+      order_3.item_orders.create(item: item_4, quantity: 10, price: item_4.price)
+      order_4.item_orders.create(item: item_5, quantity: 10, price: item_5.price)
+      order_4.item_orders.create(item: item_3, quantity: 10, price: item_3.price)
+
+      visit admin_path
+
+      within "#order-#{order_1.id}" do
+        expect(page).to have_content(order_1.id)
+        expect(page).to have_content(order_1.created_at)
+        expect(page).to have_content(order_1.status)
+        expect(page).to have_link(order_1.user.name)
+      end
+
+      within "#order-#{order_2.id}" do
+        expect(page).to have_content(order_2.id)
+        expect(page).to have_content(order_2.created_at)
+        expect(page).to have_content(order_2.status)
+        expect(page).to have_link(order_2.user.name)
+      end
+
+      within "#order-#{order_3.id}" do
+        expect(page).to have_content(order_3.id)
+        expect(page).to have_content(order_3.created_at)
+        expect(page).to have_content(order_3.status)
+        expect(page).to have_link(order_3.user.name)
+      end
+
+      within "#order-#{order_4.id}" do
+        expect(page).to have_content(order_4.id)
+        expect(page).to have_content(order_4.created_at)
+        expect(page).to have_content(order_4.status)
+        expect(page).to have_link(order_4.user.name)
+      end
+
+      within "#order-#{order_1.id}" do
+        click_link "#{order_1.user.name}"
+      end
+
```

Comment:
I think we are missing a way to actually test that the info shows up in the order that we want. 

You can use something like this to test the order it appears on the page: 
```
        expect(page.body.index(order_4.id)).to be < page.body.index(order_2.id)
        expect(page.body.index(order_2.id)).to be < page.body.index(order_3.id)
        expect(page.body.index(order_3.id)).to be < page.body.index(order_1.id)
```